windows/kolibri/LICENSE

```diff
@@ -1,20 +1,21 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2015 Learning Equality
+Copyright (c) 2016 Learning Equality
```

Comment:
Are we supposed to keep the 2016 here for some reason? Or go directly to 2017... 😛 
@jamalex @indirectlylit